public/js/components/FormFields/FormFieldTagPicker.js

```diff
@@ -0,0 +1,109 @@
+import React, {PropTypes} from 'react';
+import ShowErrors from '../Utilities/ShowErrors';
+import { errorPropType } from '../../constants/errorPropType';
+import {searchTags} from '../../services/capi';
+
+export default class FormFieldTagPicker extends React.Component {
+
+
+  static propTypes = {
+    fieldLabel: PropTypes.string,
+    fieldName: PropTypes.string,
+    fieldValue: PropTypes.string,
+    fieldPlaceholder: PropTypes.string,
+    fieldErrors: PropTypes.arrayOf(errorPropType),
+    formRowClass: PropTypes.string,
+    onUpdateField: PropTypes.func
+  };
+
+  state = {
+    suggestions: undefined,
```

Comment:
Is it better to say `undefined` of `null`? As `null` is more specifically, a blank object reference. You'd till get the advantage of falsyness, but match the type.